src/bootstrap/compile.rs

```diff
@@ -231,6 +231,18 @@ pub fn std_cargo(builder: &Builder<'_>, target: Interned<String>, cargo: &mut Ca
             }
         }
     }
+
+    // By default, rustc uses `-Cbitcode-in-rlib=yes`, and Cargo overrides that
+    // with `-Cbitcode-in-rlib=yes` for non-LTO builds. However, libstd must be
```

Comment:
I suspect one of these "yes" should be "no".

Reply:
True! Thanks, I have updated.